modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -13,9 +13,14 @@
 #include <ade/util/zip_range.hpp>
 #include <opencv2/gapi/infer.hpp>
 #include <opencv2/gapi/own/convert.hpp>
+#include <opencv2/gapi/gframe.hpp>
 
 #include "api/gbackend_priv.hpp" // FIXME: Make it part of Backend SDK!
 
+namespace {
+struct ONNXCallContext;
```

Comment:
Is ONNXCallContext also defined in an anonymous namespace?

Reply:
Yes.